internal/cmd/commands/database/init.go

```diff
@@ -546,3 +548,12 @@ func (c *InitCommand) ParseFlagsAndConfig(args []string) int {
 
 	return 0
 }
+
+func (c *InitCommand) unsetDirty(man *schema.Manager) int {
+	if err := man.UnsetDirty(c.Context); err != nil {
+		c.UI.Error(base.WrapAtLength("Database could not be marked undirty after" +
```

Comment:
Note that since you don't have a space after `after` this will end up as `...undirty afterinitialization`. Might as well just put it on one line and let WrapAtLength do its thing.